### ISSUE TITLE
Using chromecasts with a server hosted on heroku

### ISSUE THREAD

**Issue by darko1002001** [vincentbernat/dashkiosk#58]
First of all, thanks for creating this project!

I still don't know a lot about how the chromecast discovery protocols work but If i understand correctly the chromecast support would work if we host dashkiosk in the same network with the chromecasts due to the way they are discovered. So hosting on something like heroku and having chromecast support is pretty much impossible.

but how about using something like to cast the dashkiosk receiver url:
https://github.com/boombatower/chromecast-dashboard

Basically have a receiver application that loads the receiver url in a iframe or redirect to the dashkiosk screen url.

**Comment by vincentbernat** [comment]
You can use this application. Just give the URL for Dashkiosk receiver (ending with `/receiver`) to it.

**Comment by vincentbernat** [comment]
Not much to do here. I am closing this issue.
